_posts/2024-10-10-pyOpenSci-celebrates-Inessa-pawson.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: single
+title: "pyOpenSci celebrates Inessa Pawson - Community Leader"
+excerpt: "Inessa’s leadership and dedication have profoundly shaped the open source community, from NumFOCUS to NumPy and SciPy. Learn more about her impact and contributions."
+author: "Leah Wasser"
+permalink: /blog/pyopensci-celebrates-inessa-pawson.html
+header:
+  overlay_image: images/headers/pyopensci-inessa.png
+  overlay_filter: rgba(20, 13, 36, 0.3)
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+last_modified: 2024-10-10
+---
+
+# pyOpenSci Celebrates Inessa
+
+Today, I want to take a moment to celebrate a devoted pyOpenSci community member, colleague, and friend, Inessa, who was just awarded the NumFOCUS Community Leadership award. In my opinion, no one is more deserving of recognition for their contributions and dedication to the open source community. Inessa's day job is devoted to open source, where she serves as the Open Source Program Manager at [Open Teams](https://otincubator.com/team.html). But her open source work extends far beyond her professional role!
```

Comment:
```suggestion
Today, I want to take a moment to celebrate a devoted pyOpenSci community member, colleague, and friend, [Inessa](https://github.com/InessaPawson/InessaPawson), who was just awarded the NumFOCUS Community Leadership award. In my opinion, no one is more deserving of recognition for their contributions and dedication to the open source community. Inessa's day job is devoted to open source, where she serves as the Open Source Program Manager at [Open Teams](https://otincubator.com/team.html). But her open source work extends far beyond her professional role!
```

Reply:
I couldn't find a link to the award announcement, but it would be nice to link to it with the NumFOCUS Community Leadership award text.